src/certifier.cc

```diff
@@ -148,18 +140,15 @@ bool certifier::framework::policy_store::put(unsigned ent, const string v) {
               const string& type, const string& value) {
     int ent = find_entry_by_tag_and_type(tag, type);
     if (ent < 0) {
-      if (!add_entry(tag, type, value)) {
-        return false;
-      } else {
-        return true;
-      }
+      return add_entry(tag, type, value);
     }
-    put(ent, value);
+    if (!put(ent, value))
+      return false;
```

Comment:
Small request: Like it was done on L143, it will be simpler to replace this if()-else() with a simple: 

`return put(ent, value);`